engine/apps/slack/scenarios/scenario_step.py

```diff
@@ -1,79 +1,46 @@
 import importlib
 import logging
+import typing
 
 from apps.slack.alert_group_slack_service import AlertGroupSlackService
 from apps.slack.slack_client import SlackClientWithErrorHandling
 
-logger = logging.getLogger(__name__)
-
-
-PAYLOAD_TYPE_INTERACTIVE_MESSAGE = "interactive_message"
-ACTION_TYPE_BUTTON = "button"
-ACTION_TYPE_SELECT = "select"
-
-PAYLOAD_TYPE_SLASH_COMMAND = "slash_command"
-
-PAYLOAD_TYPE_EVENT_CALLBACK = "event_callback"
-EVENT_TYPE_MESSAGE = "message"
-EVENT_TYPE_MESSAGE_CHANNEL = "channel"
-EVENT_TYPE_MESSAGE_IM = "im"
-# Slack event "user_change" is deprecated in favor of "user_profile_changed".
-# Handler for "user_change" is kept for backward compatibility.
-EVENT_TYPE_USER_CHANGE = "user_change"
-EVENT_TYPE_USER_PROFILE_CHANGED = "user_profile_changed"
-EVENT_TYPE_APP_MENTION = "app_mention"
-EVENT_TYPE_MEMBER_JOINED_CHANNEL = "member_joined_channel"
-EVENT_TYPE_IM_OPEN = "im_open"
-EVENT_TYPE_APP_HOME_OPENED = "app_home_opened"
-EVENT_TYPE_SUBTEAM_CREATED = "subteam_created"
-EVENT_TYPE_SUBTEAM_UPDATED = "subteam_updated"
-EVENT_TYPE_SUBTEAM_MEMBERS_CHANGED = "subteam_members_changed"
-EVENT_SUBTYPE_MESSAGE_CHANGED = "message_changed"
-EVENT_SUBTYPE_MESSAGE_DELETED = "message_deleted"
-EVENT_SUBTYPE_BOT_MESSAGE = "bot_message"
-EVENT_SUBTYPE_THREAD_BROADCAST = "thread_broadcast"
-EVENT_TYPE_CHANNEL_DELETED = "channel_deleted"
-EVENT_TYPE_CHANNEL_CREATED = "channel_created"
-EVENT_TYPE_CHANNEL_RENAMED = "channel_rename"
-EVENT_TYPE_CHANNEL_ARCHIVED = "channel_archive"
-EVENT_TYPE_CHANNEL_UNARCHIVED = "channel_unarchive"
+if typing.TYPE_CHECKING:
+    from apps.slack.models import SlackTeamIdentity, SlackUserIdentity
+    from apps.slack.types import EventPayload
+    from apps.user_management.models import Organization, User
 
-PAYLOAD_TYPE_BLOCK_ACTIONS = "block_actions"
-BLOCK_ACTION_TYPE_USERS_SELECT = "users_select"
-BLOCK_ACTION_TYPE_BUTTON = "button"
-BLOCK_ACTION_TYPE_STATIC_SELECT = "static_select"
-BLOCK_ACTION_TYPE_CONVERSATIONS_SELECT = "conversations_select"
-BLOCK_ACTION_TYPE_CHANNELS_SELECT = "channels_select"
-BLOCK_ACTION_TYPE_OVERFLOW = "overflow"
-BLOCK_ACTION_TYPE_DATEPICKER = "datepicker"
-BLOCK_ACTION_TYPE_CHECKBOXES = "checkboxes"
-
-PAYLOAD_TYPE_DIALOG_SUBMISSION = "dialog_submission"
-PAYLOAD_TYPE_VIEW_SUBMISSION = "view_submission"
-
-PAYLOAD_TYPE_MESSAGE_ACTION = "message_action"
-
-THREAD_MESSAGE_SUBTYPE = "bot_message"
```

Comment:
`THREAD_MESSAGE_SUBTYPE` is not referenced anywhere ([search](https://github.com/search?q=repo%3Agrafana%2Foncall%20THREAD_MESSAGE_SUBTYPE&type=code))